src/hotspot/share/utilities/vmError.cpp

```diff
@@ -1803,7 +1803,7 @@ typedef void (*voidfun_t)();
 
 // Crash with an authentic sigfpe
 volatile int sigfpe_int = 0;
-static void crash_with_sigfpe() {
+static void NOINLINE crash_with_sigfpe() {
```

Comment:
Did you consider ALWAYSINLINE instead so the tests could be left unchanged?

Reply:
Used the ALWAYINLINE and tested for tier1-5.